### DISCUSSION TITLE
Pagination and change rows per page bug

### DISCUSSION THREAD

**Discussion by Cichyy22** [Power-Components/livewire-powergrid#1686] [BUG]
Hello,
I recently came across a problem with pagination in powergrid, after changing the page from the initial to page 3 and after that we increase the number of rows per page from 10 to 100, the pagination in the url will still indicate that we are on page 3 and therefore no result will be displayed to us, because in theory all rows are now available on the first page. Is there any solution to this problem?

**Comment by luanfreitasdev** [comment]
Hello. I didn't quite understand the problem, can you record a video or create a simple capture?

**Reply by Cichyy22** [reply]
> Hello. I didn't quite understand the problem, can you record a video or create a simple capture?

![sc1](https://github.com/user-attachments/assets/137face6-c88b-4507-ab30-c1e79da7bd87)
![sc2](https://github.com/user-attachments/assets/c0b0637c-e983-46e9-93d7-ecfe5b5a9fa2)
